src/components/ui/input/index.js

```diff
@@ -45,16 +47,32 @@ const Input = ({
     }
   };
 
-  return (
-    <>
+  const inputElement =
+    type === 'textarea' ? (
       <textarea
         className={`input input--${size}`}
         onChange={handleChange}
         placeholder={placeholder}
         ref={ref}
         rows={1}
         value={value}
+        data-test={dataTest}
       />
+    ) : (
+      <input
+        type={type}
+        className={`input input--${size}`}
+        onChange={handleChange}
+        placeholder={placeholder}
+        ref={ref}
+        value={value}
+        data-test={dataTest}
+      />
+    );
+
+  return (
+    <>
+      {inputElement}
```

Comment:
something not related to the PR but we have (https://github.com/kedro-org/kedro-viz/blob/main/src/styles/_extends.scss) which has `cursor: pointer` for input text boxes. For something where user needs to type in, can we remove this default ? @jitu5 @tynandebold @rashidakanchwala

Reply:
good shout .. can we create an issue and add it as quick win.